cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -36,19 +36,40 @@ Feature: Users can authneticate with OIDC authenticator
       member: !user alice
 
     - !user alice@conjur.net
-
-    - !grant
-      role: !group conjur/authn-oidc/keycloak/users
-      member: !user alice@conjur.net
     """
     And I am the super-user
     And I successfully set OIDC variables
 
   Scenario: A valid id token to get Conjur access token
-    When I successfully authenticate via OIDC
+    When I successfully authenticate via OIDC with id token:
+    """
+    {"preferred_username": "alice","email": "alice@conjur.net"}
+    """
+
```

Comment:
Can you remove this blank line plz so that the test is continuous? Same with the ones below.

Reply:
done